src/main/java/dev/tssvett/schedule_bot/bot/formatter/ScheduleStringFormatter.java

```diff
@@ -1,108 +1,106 @@
 package dev.tssvett.schedule_bot.bot.formatter;
 
 import dev.tssvett.schedule_bot.backend.dto.LessonInfoDto;
-import dev.tssvett.schedule_bot.backend.mapper.Mapper;
-import dev.tssvett.schedule_bot.persistence.entity.Lesson;
+import dev.tssvett.schedule_bot.bot.utils.message.MessageTextConstantsUtils;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Component;
 
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
 import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
 
 @Slf4j
 @Component
 public class ScheduleStringFormatter {
+    private static final List<String> DAYS_OF_WEEK = List.of("понедельник", "вторник", "среда", "четверг",
+            "пятница", "суббота", "воскресенье");
```

Comment:
```suggestion
private static final List<String> DAYS_OF_WEEK = Arrays.stream(DayOfWeek.values())
            .map(day -> day.getDisplayName(TextStyle.FULL, new Locale("ru", "RU")))
            .collect(Collectors.toList());
```
Выглядит сложней, но зато ничего в коде не захардкожено